html/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<head>
+    <title>My Personal Website</title>
+    <style></style>
+</head>
+<body>
+    <header>
+        <img src="/media/cc0-images/grapefruit-slice-332-332.jpg">
+        <div class = "quicknav">
+            <a href="#News">News</a>
+            <a href="#contact">Contact</a>
+            <a href="#about">About</a>
+          </div>
+        <div class = "searchbar">
+          <input type="text" placeholder="Search..">
+        </div>
+    </header>
+    <main>
+        <div>
+            <h2 id ="about">About</h2>
+            <p>Lorem Ipsum</p>
+            </div>
+        <article id = News>
+	        <h2>News</h2>
+	        <p>I am a paragraph…</p>
+            </article>
+        <aside>
+            <h1>Switzerland</h1>
+            <p>Switzerland, a land-locked country in the middle of geographic
+            Europe, has not joined the geopolitical European Union, though it is
+            a signatory to a number of European treaties.</p>
+           </aside>
+    </main>
+    <footer>
+        <div id="contact">
+            <a href="github.com">Github</a>
+            <a href="linkedin.com">Linkedin</a>
+            <a href="instagram.com">Instagram</a>
+          </div>
```

Comment:
```suggestion
        <ul id="contact">
            <li><a href="github.com">Github</a></li>
            <li><a href="linkedin.com">Linkedin</a></li>
            <li><a href="instagram.com">Instagram</a></li>
       </ul>
```

As you list some contact links to external websites here it is more semantic to use a `ul` element instead of a `div` element. See my suggestion. :)